### ISSUE TITLE
Switch to EE 9 XSDs + DTDs

### ISSUE THREAD

**Issue by scottmarlow** [eclipse-ee4j/glassfish#23085]
In the [Jakarta EE 9 Platform TCK nightly build](https://ci.eclipse.org/jakartaee-tck/job/jakartaeetck-nightly-build-master/), we are trying to run the samples tests but seem to get a failure with the glassfish6/glassfish/lib/schemas/connector_1_7.xsd & glassfish6/glassfish/lib/dtds/connector_1_5.dtd, mentioned in comment [jakartaee-tck/pull/292](https://github.com/eclipse-ee4j/jakartaee-tck/pull/292#discussion_r437310595).

Please switch to EE 9 XSDs, or at least the EE 9 connector XSD, some links are [https://jakarta.ee/xml/ns/jakartaee/connector_2_0.xsd](https://jakarta.ee/xml/ns/jakartaee/connector_2_0.xsd)
[https://github.com/eclipse-ee4j/jakartaee-schemas/blob/master/jakartaee9/src/connector_2_0.xsds](https://github.com/eclipse-ee4j/jakartaee-schemas/blob/master/jakartaee9/src/connector_2_0.xsds)

We should also switch to EE 9 DTDs.

## Environment Details
* GlassFish Version (6) [engineering build](https://ci.eclipse.org/jakartaee-tck/job/build-glassfish/).
* JDK version: 8
* OS: Linux
* Database: Derby

**Comment by smillidge** [comment]
Yeah we haven't updated the schemas yet. I didn't even know this one had been done. 
Can you point to a Ear or war that easily reproduces this?

**Comment by smillidge** [comment]
The error in the TCK pull request looks like a parsing error. Do you know how I get the rar file trying to be deployed.

**Comment by smillidge** [comment]
https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/tests/common/connector/whitebox/permissiondd/ra.xml still references the old deployment descriptor. 

I think all we need to do is package the updated xsd as I don't think GlassFish pulls in the xsd explicitly it just parses the file.

**Comment by scottmarlow** [comment]
Thanks @smillidge!  I'll give that a try and close this after verifying.

**Comment by smillidge** [comment]
May as well leave open as we don need to package the new xsds

**Comment by smillidge** [comment]
Opened #23086 to package the latest schemas
